### DISCUSSION TITLE
SHU-Cyberspace-Security-101/6_forum/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#98] [General]
# SHU-Cyberspace-Security-101/6_forum/

SHU Cyberspace Security 101

https://shu-cake1salie.github.io/SHU-Cyberspace-Security-101/6_forum/

**Comment by LuminolT** [comment]
沙发【？

**Reply by TonyCrane** [reply]
草，comment 的链接、id 啥的忘改了吧兄弟

**Reply by LuminolT** [reply]
卧槽，对的哥们儿，麻烦删一下 :-(

**Reply by TonyCrane** [reply]
看起来已经修了

我们这个写的挺针对我们自己用的，你要是搬过去的话多注意一下有没有其他地方像这里一样硬编码了

加油（

**Reply by LuminolT** [reply]
TY，非常好项目，爱来自shushu :D
